libsolidity/ast/Types.cpp

```diff
@@ -1855,12 +1834,36 @@ bool ArrayType::isDynamicallyEncoded() const
 	return isDynamicallySized() || baseType()->isDynamicallyEncoded();
 }
 
-bigint ArrayType::storageSizeUpperBound() const
+bigint ArrayType::sizeUpperBound(DataLocation _loc) const
```

Comment:
Thinking about this for a second: Actually this is a ``sizeLowerBound``, isn't it :-D?